content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx

```diff
@@ -8,23 +8,21 @@ preview: true
 
 <TopBlock>
 
-Prisma allows configuration of database indexes, unique constraints and primary key constraints through the `extendedIndexes` preview feature in version `3.5.0` and later. This provides extended syntax options in the Prisma schema language for configuring the length of indexes (MySQL only), the sort order (all databases), the index type (PostgreSQL only) and to configure indexes as clustered or non-clustered (SQL Server only).
+Prisma allows configuration of database indexes, unique constraints and primary key constraints. This is in General Availability in versions `4.0.0` and later. You can enable this with the `extendedIndexes` Preview feature in versions `3.5.0` and later.
```

Comment:
This change loses the information what you can actually configure. Fine if intentional.

Reply:
Yes this is intentional, I'd rather give the information in easier-to-read form further down rather than including it in the TopBlock